README.md

```diff
@@ -1,6 +1,7 @@
 # Postman Collections
 
-This repository holds a series of postman collections demonstrating a variety of use cases for all of the  **HERE REST and Platform Extension APIs**. More information about our APIs can be found on [developer.here.com](https://developer.here.com/).
+This repository holds a series of postman collections demonstrating a variety of use cases for all of the  **HERE REST and Platform Extension APIs** and are updated in sync with changes to the respective APIs.
```

Comment:
Nit (even if not part of this change), there is a double-space in "the  **HERE". Also, while at it, is there a direct link to the docs for the "HERE REST and Platform Extension APIs" that we could add here?